devices/bmp180/bmp180.go

```diff
@@ -0,0 +1,258 @@
+// Copyright 2016 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+// Package bmp180 controls a Bosch BMP180 device over I²C.
+//
+// Datasheet
+//
+// https://cdn-shop.adafruit.com/datasheets/BST-BMP180-DS000-09.pdf
+package bmp180
+
+import (
+	"fmt"
+	"time"
+
+	"periph.io/x/periph/conn"
+	"periph.io/x/periph/conn/i2c"
+	"periph.io/x/periph/devices"
+)
+
+// Oversampling affects how much time is taken to measure pressure.
+type Oversampling uint8
+
+// Possible oversampling values.
+const (
+	No  Oversampling = 0
+	O2x Oversampling = 1
+	O4x Oversampling = 2
+	O8x Oversampling = 3
+)
+
+// Dev is an handle to a bmp180.
+type Dev struct {
+	d  conn.Conn
+	c  calibration
+	os Oversampling
+}
+
+func (d *Dev) String() string {
+	return fmt.Sprintf("BMP180{%s}", d.d)
+}
+
+const (
+	ctrlCommandTemperature = 0x0E
```

Comment:
Frankly it'd make more sense if the constants were fixed here, e.g.
```
ctrlCommandTemperature = 0x0E << ctrlMeasurementControl
```

but since these commands cannot run without the start conversion bit, I don't see value in having these constant separated.